### PR TITLE
Jenkins: exclude promotion for Jenkins plugins generated images 

### PR DESCRIPTION
**Pull request by akram**
fix non promotion for Jenkins plugins to allow automatic ci-config upgrade

### CODE REVIEW

**Comment by waveywaves** [comment]
can you do a `make jobs` and push again ?

File: ci-operator/config/openshift/jenkins-client-plugin/openshift-jenkins-client-plugin-master.yaml [hunk 1]
```diff
@@ -22,6 +22,10 @@ images:
         source_path: /go/src/github.com/openshift/jenkins-client-plugin/.
   to: jenkins
 
+promotion:
+  excluded_images:
+  - jenkins
```
Comment (petr-muller) [inline ci-operator/config/openshift/jenkins-client-plugin/openshift-jenkins-client-plugin-master.yaml]: The promotion also needs a destination, so this should be:

```
promotion:
  name: "4.3"
  namespace: ocp
  excluded_images:
  - jenkins
```

File: ci-operator/config/openshift/jenkins-openshift-login-plugin/openshift-jenkins-openshift-login-plugin-master.yaml [hunk 2]
```diff
@@ -22,6 +22,10 @@ images:
         source_path: /go/src/github.com/openshift/jenkins-openshift-login-plugin/.
   to: jenkins
 
+promotion:
```
Comment (petr-muller) [inline ci-operator/config/openshift/jenkins-openshift-login-plugin/openshift-jenkins-openshift-login-plugin-master.yaml]: Given you bumped the `4.1` to `4.3` in the other two files, maybe you should do it here too?

**Comment by akram** [comment]
should be better @petr-muller  now

**Pull request merged**